app/assets/stylesheets/components/_auth-form.scss

```diff
@@ -0,0 +1,80 @@
+.auth-form {
+  width: 100%;
+  max-width: 450px;
+  margin: auto 0;
+
+  &__wrapper {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 40px 50px 20px;
+    border-radius: 10px;
+    background-color: $white-500;
+    box-shadow: 0 0 10px #dbdbdb, 0 0 10px #fff;
+  }
+
+  &__top,
+  &__bot {
+    width: 100%;
+  }
+
+  &__top {
+    margin-bottom: 50px;
+  }
+
+  &__bot {
+    form {
+      margin-bottom: 60px;
+    }
+  }
+
+  &__field {
+    display: flex;
+    flex-direction: column;
+    margin-bottom: 20px;
+
+    input {
```

Comment:
Should style this input as the base style?
Or this will be totally different from the other input in the future?

Reply:
Yup I centralized a few other styles too that I spotted. Good catch 😄 Updated in https://github.com/aetheryna/google-scraper-ruby/pull/32/commits/6e5051013e49fbd4fc6fe209042953850e62b12d